### PR TITLE
Revise readme

### LINKED ISSUE #22: Extend README.md
README.md is transferred and shown on the pypi page. We should add more information and a link to the documentation until the next release.

### PR DESCRIPTION
**Pull request by hdsassnick**
Closes #22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+# aim2dat
 
-Python library for pre-, post-processing and data management of ab-initio high-throughput workflows for computational materials science.
+aim2dat (Automated Ab-Initio Materials Modeling and Data Analysis Toolkit) is a library for pre-, post-processing and data management of ab-initio high-throughput workflows for computational materials science.
+For further details and documentation, please visit https://aim2dat.github.io/aim2dat.
+
+## Feature List
+
+* Managing and analysing sets of crystals and molecules.
+* Ab-initio high-throughput calculations based on aiida.
```
Comment (t-reents) [inline README.md:9]:
```suggestion
* Ab-initio high-throughput calculations based on AiiDA.
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+* Plotting material's properties such as electronic band structures, projected density of states or phase diagrams.
+* Interface to machine learning routines via sci-kit learn.
```
Comment (t-reents) [inline README.md:11]:
```suggestion
* Interface to machine learning routines via [sci-kit learn](https://scikit-learn.org/stable/).
```

**Review by t-reents — APPROVED** [review]
I added small suggestions, e.g. to add the links to the corresponding packages.

Moreover, I think we should add a small section `How to contribute` (or a different naming like `Development`) at some point, containing some instructions on using `pytest` and `pre-commit`. However, this can be also done at a later stage. Just as a general comment

**Comment by hdsassnick** [comment]
Thanks @t-reents , I added the Contributing section and adjusted some minor stuff.

**Review by t-reents — APPROVED** [review]
Looks good, thanks for the changes @hdsassnick !

**Review by Scemp — APPROVED** [review]
It sums everything very well up, and I have no further suggestions!

**Pull request merged**